examples/bri-3/src/bri/zeroKnowledgeProof/api/dtos/request/verifyAnchorHash.dto.ts

```diff
@@ -0,0 +1,13 @@
+import { IsNotEmpty } from 'class-validator';
+import { ZeroKnowledgeProofVerificationInput } from '../../../models/zeroKnowledgeProofVerificationInput';
+import { DocumentObject } from '../../../models/document';
+
+export class VerifyAnchorHashDto {
+  @IsNotEmpty()
+  inputForProofVerification:
+    | DocumentObject
+    | ZeroKnowledgeProofVerificationInput;
+
+  @IsNotEmpty()
+  signature: string;
```

Comment:
are we checking somewhere for the signature regex?

Reply:
Not yet. We can add validations in the upcoming PRs